examples/webgpu-embedding-benchmark/main.js

```diff
@@ -10,10 +10,8 @@ if (!navigator.gpu) {
 }
 
 // Proxy the WASM backend to prevent the UI from freezing
```

Comment:
```suggestion
```

As `env.backends.onnx.wasm.proxy = true;` has been moved away from here (moved to `src/backends/onnx.js`), this comment-line can also be removed.

Reply:
Good catch - thanks!